src/generics/constants/api-responses.js

```diff
@@ -6,5 +6,6 @@
  */
 
 module.exports = {
-    UPDATE_QUERY_REQUIRED: "UPDATE_QUERY_REQUIRED"
+    UPDATE_QUERY_REQUIRED: "UPDATE_QUERY_REQUIRED",
+    PROJECT_NOT_FOUND:"good result"
```

Comment:
@MallanagoudaB Replace PROJECT_NOT_FOUND with Some relevant message. And the message should be proper like entity type not found. Change the message according to the error